bedrock/products/templates/products/vpn/platforms/desktop.html

```diff
@@ -0,0 +1,164 @@
+{# This Source Code Form is subject to the terms of the Mozilla Public
+ # License, v. 2.0. If a copy of the MPL was not distributed with this
+ # file, You can obtain one at http://mozilla.org/MPL/2.0/. -#}
+
+{% set vpn_devices = 5 %}
+{%  set vpn_servers = 280 %}
```

Comment:
Is it worth doing this for countries as well?

Nit: move these below `{% extends "products/vpn/base.html" %}`